10-setup-htcondor.sh

```diff
@@ -30,16 +35,18 @@ if [ "x$ANNEX_NAME" = "x" ]; then
     export ANNEX_NAME="$GLIDEIN_ResourceName@$GLIDEIN_Site"
 fi
 
-# token auth
-mkdir -p ~/.condor/tokens.d
-echo "$TOKEN" >~/.condor/tokens.d/flock.opensciencegrid.org
-chmod 600 ~/.condor/tokens.d/flock.opensciencegrid.org
-mkdir -p ~/.condor/passwords.d
-chmod 700 ~/.condor/passwords.d
-
-# glorious hack
-export _CONDOR_SEC_PASSWORD_FILE=~/.condor/tokens.d/flock.opensciencegrid.org
-export _CONDOR_SEC_PASSWORD_DIRECTORY=~/.condor/passwords.d
```

Comment:
@rynge do you recall the reasoning behind this glorious hack? Why not throw these config settings directly into the condor config that are generated below?

Reply:
Todd Milled added that, but I suspect it was for versions earlier in the 8.9 series. I want to do some testing before approving this PR.

Reply:
`_CONDOR_SEC_PASSWORD_DIRECTORY` in a non-system location was to support running the container under a regular user, for example in the case of Singularity.

Reply:
Good point. @edquist let's use the non-system locations when we copy the token in from a volume. @rynge what should the ownership be to support the singularity and non-singularity use cases?

Reply:
The ownership is tricky as that depends on what the site/user wants. Think of this as a pilot. For example, some site will have it as `osg` with some uid, some other site with a different username and different uid. I think this needs to be dynamic, and probably any non-0 uid should be acceptable.